pkg/test/kube/dump.go

```diff
@@ -271,6 +275,12 @@ func DumpPodEvents(_ resource.Context, c cluster.Cluster, workDir, namespace str
 			return
 		}
 
+		for i := range list.Items {
```

Comment:
Can you clarifying what you're doing here an way in a comment? I'm assuming the managed fields bloat the dump?